electrolyzer/supervisor.py

```diff
@@ -137,6 +141,21 @@ def update_stack_status(self):
                 self.waiting[i] = 0
                 self.stacks_waiting_vec[i] = 0
 
+    def initialize_plant_stacks(self):
```

Comment:
I know we don't have any explicit `Supervisor` tests, but we might want a basic test for this in `test_run_electrolyzer.py`. It could just be a super short simulation where we assert that everything starts up as expected

Reply:
@camirmas I can work on writing up a simple test of this initialization function
Do we want to just test this function, or both the start-up options?

Reply:
It's up to you, we have a few basic integration tests in that file already so I think it's fine to just do one integration test that makes sure that this initialization case works.